### ISSUE TITLE
Scale above 0dB

### ISSUE THREAD

**Issue by jopoet** [vvvar/PeakEater#64]
Would be great to have an option to show waveform above 0dB up to +3dB or +6dB.

**Comment by vvvar** [comment]
Yeah then it may look more similar to other plugins(this scaling is more-or-less a visual representation), thanks for feedback, will have a look what could be done.
